test/e2e/README.md

```diff
@@ -30,6 +30,28 @@ Multiple testnets can be run with the `run-multiple.sh` script:
 ./run-multiple.sh networks/generated/gen-group3-*.toml
 ```
 
+Testnets running different versions of Tendermint can be generated by the
+generator. For example:
+
+```sh
+# Generate testnets randomly choosing between v0.34.21 (with a random weighting
+# of 1) and v0.34.22 (with a random weighting of 2).
+./build/generator -m "v0.34.21:1,v0.34.22:2" -d networks/generated/
+
+# "HEAD" refers to the code on the current branch.
+./build/generator -m "v0.34.23:1,HEAD:2" -d networks/generated/
+
+# Using "latest" will cause the generator to auto-detect the latest
+# non-pre-release version tag in the current Git repository that is closest to
+# the Tendermint version in this repository (as specified in
+# ../../version/version.go).
```

Comment:
Would it make sense to only consider the releases of the current git branch going backwards in time? (rather than all branches)

Reply:
Forget about this comment. I've been doing some research and some small experiments and what I'm suggesting is probably too complex, along the lines of implementing `git describe` (example [here](https://github.com/edupo/semver-cli/blob/master/gitWrapper/git.go#L44))